src/ResultManager.js

```diff
@@ -1,10 +1,22 @@
+import Result from './Result.js';
+
 export default class ResultManager {
   constructor({ results = [] } = {}) {
     this.results = results;
   }
 
-  addResult(Result) {
-    this.results.push(Result);
+  addResult(result = {}) {
```

Comment:
- [ ] As mentioned above Let's use `arrow functions` instead of the standard functions.